Contributing.md

```diff
@@ -0,0 +1,212 @@
+# Contribute to this documentation
+
+Thank you for your interest in our documentation!
+
+* [Ways to contribute](#ways-to-contribute)
+* [Contribute using GitHub](#contribute-using-github)
+* [Contribute using Git](#contribute-using-git)
+* [How to use Markdown to format your topic](#how-to-use-markdown-to-format-your-topic)
+* [FAQ](#faq)
+* [More resources](#more-resources)
+
+## Ways to contribute
+
+Here are some ways you can contribute to this documentation:
+
+* To make small changes to an article, [Contribute using GitHub](#contribute-using-github).
+* To make large changes, or changes that involve code, [Contribute using Git](#contribute-using-git).
+* Report documentation bugs via GitHub Issues
+* Request new documentation at the [Office Developer Platform UserVoice](http://officespdev.uservoice.com) site.
+
+## Contribute using GitHub
+
+Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. Use this method to make a minor change that doesn't involve code changes. 
+
+**Note** Using this method allows you to contribute to one article at a time.
+
+### To Contribute using GitHub
+
+1. Find the article you want to contribute to on GitHub. 
+
+    If the article is in MSDN, choose the **suggest and submit changes** link in the **Contribute to this content** section and you'll be taken to the same article on GitHub.
+2. Once you are on the article in GitHub, sign in to GitHub (get a free account [Join GitHub](https://github.com/join).
+3. Choose the **pencil icon** (edit the file in your fork of this project) and make your changes in the **<>Edit file** window. 
+4. Scroll to the bottom and enter a description.
+5. Choose **Propose file change**>**Create pull request**.
+
+You now have successfully submitted a pull request. Pull requests are typically reviewed within 10 business days. 
```

Comment:
Is 10 business days the official policy?

Reply:
As explained in my prior comment, the information in this file is standard info that is used across all OfficeDev GitHub repos. Therefore, I have no reason to believe that it's not correct -- but I'll make a note to confirm @davidchesnut (once he's back from leave).